SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -27,11 +29,22 @@ void USpatialPlayerSpawner::Init(USpatialNetDriver* InNetDriver, FTimerManager*
 	NumberOfAttempts = 0;
 }
 
-void USpatialPlayerSpawner::ReceivePlayerSpawnRequest(FString URLString, const char* CallerAttribute, Worker_RequestId RequestId )
+void USpatialPlayerSpawner::ReceivePlayerSpawnRequest(Schema_Object* Payload, const char* CallerAttribute, Worker_RequestId RequestId )
 {
+	// Extract spawn parameters.
```

Comment:
Worth adding a comment to which component this is referring to?

Reply:
Not a component. It's the `PlayerSpawnRequest`, so I figured the function name `ReceivePlayerSpawnRequest` should be self-documenting enough?